### DISCUSSION TITLE
Can I use sqlite on vercel functions alongside prisma?

### DISCUSSION THREAD

**Discussion by LGmatrix13** [vercel/vercel#6078] [Help] [answered]
Hello all,

I'm using Prisma alongside SQLite and NextJS in a project. Is it permissible to use with vercel? I can easily switch it for Postgres with Prisma, but that requires some extra cost/usage of time.

Thanks!

**Comment by styfle** [comment, marked as answer]
Hi @LGmatrix13 

No, not possible. Unfortunately, sqlite is designed for local DBs, not Serverless.

Take a look at #5327 and #4556 and #4443.

**Reply by LGmatrix13** [reply]
Understood. I’ll look into Postgres with DO so my frontend/api can remain on Vercel.

**Reply by styfle** [reply]
Also take a look at this document: https://vercel.com/docs/solutions/databases

**Reply by LGmatrix13** [reply]
@styfle Thank you
